cmd/snap/cmd_debug_seeding_test.go

```diff
@@ -394,6 +406,19 @@ seed-completion:  -
 `[1:],
 			comment: "not preseeded, still seeding",
 		},
+		{
+			jsonResp: seedingError,
+			expStdout: `
+seeded:  false
+seed-error: |
+  cannot perform the following tasks:
+  - xxx
+preseeded:         true
+image-preseeding:  9.318s
+seed-completion:   -
```

Comment:
just a note that this will conflict / break with my PR depending on which PR lands first